nixos/modules/services/desktops/cpupower-gui.nix

```
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.cpupower-gui;
+in {
+  options = {
+    services.cpupower-gui = {
+      enable = mkEnableOption "cpupower-gui";
+      description = ''
+        Enables dbus/systemd service needed used by cpupower-gui
+        to retrieve and modify cpu power saving settings.
+      '';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ config.boot.kernelPackages.cpupower-gui ];
+    services.dbus.packages = [ pkgs.linuxPackages.cpupower-gui ];
+    systemd.packages = [ pkgs.linuxPackages.cpupower-gui ];
```

shoudl also be changed
```suggestion
    services.dbus.packages = [ config.boot.kernelPackages.cpupower-gui ];
    systemd.packages = [ config.boot.kernelPackages.cpupower-gui ];
```

---

sorry, done now